CMakeFilters.cmake

```diff
@@ -9,14 +9,23 @@
 # If you do not have access to either file, you may request a copy from
 # help@hdfgroup.org.
 #
-option (USE_LIBAEC_STATIC "Use static AEC library " OFF)
+option (USE_ZLIB_NG "Use zlib-ng library as zlib library" OFF)
```

Comment:
Seems like we should probably be prefixing these options as in `HDF5_USE_ZLIB_NG`

Reply:
Yes - good point